setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 from os import path
-from version import get_version, update_release_version
+from ctapipe_io_lst.version import get_version, update_release_version
```

Comment:
This breaks the installation in a fresh environment, please revert.

There was a reason, why the version code, unfortunately, had to be duplicated.